Missing dependency towards md5.js

**Describe the bug**
When running this in Node.js using JavaScript and CommonJS modules, I get the following runtime error.

```
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module 'md5.js'
Require stack:
- .../node_modules/media-stream-library/dist/cjs/components/auth/digest.js
```

I think the `md5.js` `devDependencies` should be moved to `dependencies`. This will solve the issue for Node.js, but I don't know whether it will cause another issue for other runtimes.

**Environment (please complete the following information):**
 - OS: iOS
 - Browser Node.js v12.16.2
 - Version 7.0.3

--- a/package.json
+++ b/package.json
@@ -40,6 +40,7 @@
   ],
   "dependencies": {
     "debug": "4.1.1",
+    "md5.js": "1.3.5",
     "ws": "7.2.5"
   },
   "devDependencies": {
@@ -66,7 +67,6 @@
     "eslint-plugin-promise": "4.2.1",
     "http-server": "0.12.3",
     "jest": "25.5.4",
-    "md5.js": "1.3.5",
     "mock-socket": "9.0.3",
     "prettier": "2.0.5",
     "ts-jest": "25.4.0",

